templates/blockonomics_checkout.php

```diff
@@ -9,136 +9,64 @@
-            <span class="bnomics-order-number" ng-cloak> <?=__('Order #', 'blockonomics-bitcoin-payments')?>{{order.order_id}}</span>
+            <span class="bnomics-order-number"><?php echo 'ORDER #' . $order['order_id']?></span>
           </div>
         </div>
       </div>
-      <!-- Spinner -->
-      <div class="bnomics-spinner" ng-show="spinner" ng-cloak><div class="bnomics-ring"><div></div><div></div><div></div><div></div></div></div>
       <!-- Amount row -->
       <div class="bnomics-order-panel">
         <div class="bnomics-order-info">
 
-          <div class="bnomics-bitcoin-pane" ng-hide="show_altcoin != 0" ng-init="show_altcoin=0;" ng-cloak>
+          <div class="bnomics-bitcoin-pane">
             <div class="bnomics-btc-info">
-              <!-- QR and Amount -->
-              <div class="bnomics-qr-code" ng-hide="order.status == -3">
-                <div class="bnomics-qr">
-                          <a href="bitcoin:{{order.address}}?amount={{order.satoshi/1.0e8}}">
-                            <qrcode data="bitcoin:{{order.address}}?amount={{order.satoshi/1.0e8}}" size="160" version="6">
-                              <canvas class="qrcode"></canvas>
-                            </qrcode>
-                          </a>
-                </div>
-                <div class="bnomics-qr-code-hint"><?=__('Click on the QR code to open in the wallet', 'blockonomics-bitcoin-payments')?></div>
-              </div>
               <!-- BTC Amount -->
               <div class="bnomics-amount">
               <div class="bnomics-bg">
                 <!-- Order Status -->
                 <div class="bnomics-order-status-wrapper">
-                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'blockonomics-bitcoin-payments')?></span>
-                  <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('<b>PAYMENT EXPIRED</b> <br /><br /><a href="javascript:history.back()">Click here</a> to try again.<br /><br /><div>If you already paid, your order will be processed automatically. <br />You can safely close this window.</div>', 'blockonomics-bitcoin-payments')?></span>
-                  <span class="warning bnomics-status-warning" ng-show="order.status == -2" ng-cloak><?=__('Payment Error', 'blockonomics-bitcoin-payments')?></span>
+                  <span class="bnomics-order-status-title"><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'blockonomics-bitcoin-payments')?></span>
                 </div>
-                    <h4 class="bnomics-amount-title" for="invoice-amount" ng-hide="order.status == -3">
-                     {{order.satoshi/1.0e8}} BTC
+                    <h4 class="bnomics-amount-title" for="invoice-amount">
+                     <?php echo $order['satoshi']/1.0e8;?> BTC
```

Comment:
When satoshi is very small, php is showing scientific notation https://prnt.sc/sitb04 . Format it so that it always display in human notation